solution/backend/serverless.yml

```diff
@@ -263,6 +263,10 @@ resources:
           search_path: '"$user",public'
           pgaudit.role: "rds_pgaudit"
           pgaudit.log: "ALL"
+          pgaudit.log_level: "LOG"
+          pgaudit.log_parameter: 1
+          pgaudit.log_statement_once: 0
+          pgaudit.log_catalog: 1
```

Comment:
Changes for this ticket -> [EREGCSC-2470](https://jiraent.cms.gov/browse/EREGCSC-2470)